packages/react-jss/src/types/index.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+import type {StyleSheetOptions} from 'jss'
+import jss from '../jss'
+
+export type Options = {
+  theming?: {
+    themeListener: any => any
+  },
+  inject?: Array<'classes' | 'themes' | 'sheet'>,
+  jss?: jss,
+  ...$Shape<StyleSheetOptions>
+}
+export type Theme = {}
+export type Styles = {[string]: {}}
+export type ThemerFn = (theme: Theme) => Styles
+export type StylesOrThemer = Styles | ThemerFn
+export type Classes<S> = {|
```

Comment:
Should be also imported from `jss` https://github.com/cssinjs/jss/blob/master/packages/jss/src/index.js#L33

Reply:
Fixed